packages/store-ui/src/atoms/Input/Input.stories.mdx

```diff
@@ -0,0 +1,79 @@
+import { Meta, Story, Preview, ArgsTable } from '@storybook/addon-docs/blocks'
+import Input from '.'
+
+<Meta title="Atoms/Input" />
+
+export const colorByState = {
+  default: 'black',
+  success: 'green',
+  error: 'red',
+}
+
+# Default
+
+<Preview>
+  <Story
+    name="Default"
+    args={{
+      placeholder: 'First name',
+    }}
+  >
+    {(args) => (
+      <Input
+        style={{
+          borderColor: colorByState[args.state],
+          borderStyle: 'solid',
+        }}
```

Comment:
I don't like the way storybook presents the styles when we use `style`. I think we should look into style and accessibility add-ons, but that is ok for now.

Reply:
Hmm, wanna talk about this later? I'm thinking about what we can improve there as well, but I didn't understand what do you mean by "the way storybook presents the styles".